doc/conf.py

```diff
@@ -45,7 +45,7 @@ def is_release_eol(codename):
 
 # project information
 project = 'Ceph'
-copyright = ('2016, Ceph authors and contributors. '
+copyright = ('2025, Ceph authors and contributors. '
```

Comment:
it's not necessary to update an existing file just because we are moving to a new calendar year. in general, the date should reflect the date when the file was created and modified.

Reply:
@tchaikov, Are we cool with closing this?

Reply:
this is shown on every documentatino page. 
![image](https://github.com/user-attachments/assets/82903901-c0e1-4e94-a7e5-c152d7d72e44)

Reply:
looking at https://github.com/ceph/ceph/pull/22053#discussion_r188929473 maybe something like this?
```suggestion
copyright = (f'2014-{datetime.datetime.now().year}, Ceph authors and contributors. '
```

Reply:
please also see https://en.wikipedia.org/wiki/Copyright_notice#Reasons_to_include_an_optional_copyright_notice, i am quoting the related line:

> Use of the notice informs the public that a work is protected by copyright, identifies the copyright owner, and shows the year of first publication.